tips/06-08-2017/React.xml

```diff
@@ -316,4 +316,11 @@
       <option name="JAVA_SCRIPT" value="true" />
     </context>
   </template>
-</templateSet>
+  <template name="onc" value="onClick={$END$} " description="onClick event props" toReformat="true" toShortenFQNames="true">
+    <context>
+      <option name="JAVA_SCRIPT" value="true" />
+      <option name="JS_EXPRESSION" value="false" />
+      <option name="JS_STATEMENT" value="false" />
+    </context>
+  </template>
+</templateSet>
```

Comment:
this file is generated by WebStorm, are you sure it's needed to fix it every time?